dotnet/src/SemanticKernel.Abstractions/TemplateEngine/PromptTemplateConfig.cs

```diff
@@ -14,6 +14,11 @@ namespace Microsoft.SemanticKernel.TemplateEngine;
 /// </summary>
 public class PromptTemplateConfig
 {
+    /// <summary>
+    /// Semantic Kernel template format.
+    /// </summary>
+    public static readonly string SEMANTICKERNEL = "semantic-kernel";
```

Comment:
Is there a special reason for Upper casing this "constant" ?

Follow the other examples in PascalCasing.

```suggestion
    public const string SemanticKernelTemplateFormat = "semantic-kernel";
```